conf/functions_worker.yml

```diff
@@ -125,6 +125,15 @@ functionRuntimeFactoryConfigs:
 #  jobNamespace:
 #  # the docker image to run function instance. by default it is `apachepulsar/pulsar`
 #  pulsarDockerImageName:
+#  # the docker image to run java function instance, we will uses `pulsarDockerImageName` to run function instance
```

Comment:
If the eventual goal is to make language runtimes pluggable, can we organize this into a map, something like:

`
functionDockerImages:
  JAVA: apache/pulsar:java
  PYTHON: apache/pulsar:python
  FOO: apache/pulsar:foo
  DEFAULT: apache/pulsar:default
`

Since language runtimes will be pluggable it doesn't make sense to have concrete configs for a specific language.

Reply:
Thanks @jerrypeng the comments have been fixed, PTAL again, thanks